plugins/chat/assets/javascripts/discourse/models/chat-message.js

```diff
@@ -94,6 +95,10 @@ export default class ChatMessage {
     if (args.thread) {
       this.thread = args.thread;
     }
+
+    this.site = getOwner(this).lookup("service:site");
+    this.siteSettings = getOwner(this).lookup("service:site-settings");
+    this.usersApi = getOwner(this).lookup("service:usersApi");
```

Comment:
I wonder if we need to instantiate for each message if we are not going to use it each time